packages/sanity/src/core/releases/components/BundleMenuButton/BundleMenuButton.tsx

```diff
@@ -74,6 +82,12 @@ export const BundleMenuButton = ({disabled, bundle, documentCount}: BundleMenuBu
       ...bundle,
       archivedAt: isBundleArchived ? undefined : new Date().toISOString(),
     })
+
+    if (isBundleArchived) {
+      telemetry.log(UnarchivedRelease)
```

Comment:
Trying to reason about this code. It might just be that I'm confused, but will this not log an `UnarchivedRelease` event when deleting an archived release and, below, log an `ArchivedRelease` event when deleting an unarchived release? Apologies if I'm mistaken, or this is the intention.

Reply:
I know what you mean I also felt a bit confused. This is coming from the bundle object at the top which is not updated fast enough to where this section of the code.

So technically it's:
- IsBundleArchived `true` -> it means that it's in the process of becoming false, so the event we want to track is unarchive
- IsBundleArchived `false` -> if means that it's in the process of becoming true, so the event we want to track is archive

Even the  `achivedAt: isBundleArchived ? undefined : new Date().toISOString(),` follows the same type of logic. If it's true then set it as undefined because technically it won't be archived anymore, if it's _not_ archived then add the date because it means it'll become archived

I'll add this comment there, unless you feel there is a better way? 🤔

Reply:
@juice49 I'm closing this, moved the commits to here https://github.com/sanity-io/sanity/pull/7460 :)